cmscommon/datetime.py

```diff
@@ -227,7 +227,7 @@ def monotonic_time():
         from win32api import GetTickCount
 
         def monotonic_time():
-            return GetTickCount / 1000.0
+            return GetTickCount / 1000
```

Comment:
Apart from the lack of usefulness of this branch, I'd say don't touch code borrowed from somewhere else.

Reply:
Undone.